### PR TITLE
Set name for long lived thread

### PR DESCRIPTION
**Pull request by solotzg**
Signed-off-by: Tong Zhigao <tongzhigao@pingcap.com>

### CODE REVIEW

**Comment by solotzg** [comment]
@hanfei1991 PTAL

File: src/pd/Client.cc [hunk 1]
```diff
@@ -169,6 +169,8 @@ void Client::updateURLs(const ::google::protobuf::RepeatedPtrField<::pdpb::Membe
 
 void Client::leaderLoop()
 {
+    pingcap::SetThreadName("LeaderLoopCltC");
```
Comment (hanfei1991) [inline src/pd/Client.cc:172]: adding a PD prefix is clearer ?

File: include/pingcap/pd/Oracle.h [hunk 2]
```diff
@@ -54,6 +54,8 @@ class Oracle
 private:
     void updateTS(std::chrono::milliseconds update_interval)
     {
+        pingcap::SetThreadName("UpdateTSCltC");
```
Comment (hanfei1991) [inline include/pingcap/pd/Oracle.h:57]: add a PD prefix ?

**Review by hanfei1991 — APPROVED** [review]
LGTM

**Pull request merged**